README.md

```diff
@@ -12,14 +12,14 @@ With Gradle from repo.spring.io or Maven Central repositories (stable releases o
 
 ```groovy
     repositories {
-	//maven { url 'https://repo.spring.io/snapshot' }
-	maven { url 'https://repo.spring.io/milestone' }
-	mavenCentral()
+        //maven { url 'https://repo.spring.io/snapshot' }
+        maven { url 'https://repo.spring.io/milestone' }
+        mavenCentral()
     }
 
     dependencies {
-      //compile "io.projectreactor.kotlin:reactor-kotlin-extensions:1.2.0-SNAPSHOT"
-      compile "io.projectreactor.kotlin:reactor-kotlin-extensions:1.2.0-M1"
+        //implementation "io.projectreactor.kotlin:reactor-kotlin-extensions:1.2.0-SNAPSHOT"
+        implementation "io.projectreactor.kotlin:reactor-kotlin-extensions:1.2.0-M1"
```

Comment:
Also changed to use recommanded `implementation` instead of deprecated `compile`